windows/src/developer/TIKE/child/UfrmKeymanWizard.dfm

```diff
@@ -953,11 +963,45 @@ inherited frmKeymanWizard: TfrmKeymanWizard
               TabOrder = 2
               OnClick = chkLayoutDisplay102KeyClick
             end
+            object panWarnMixedShiftStates: TPanel
+              Left = 492
+              Top = 59
+              Width = 317
+              Height = 54
+              BevelOuter = bvLowered
+              Color = clCream
+              ParentBackground = False
+              TabOrder = 5
+              Visible = False
+              object lblWarnMixedShiftStates: TLabel
+                Left = 8
+                Top = 8
+                Width = 183
+                Height = 39
+                Caption = 
+                  'Warning: mixing left/right and non-specific modifiers means some' +
+                  ' rules are not visible here'
+                WordWrap = True
+              end
+              object cmdFixupShiftStates: TButton
+                Left = 200
+                Top = 16
+                Width = 107
+                Height = 25
+                Caption = 'Fixup Shift States'
```

Comment:
Do you want a hotkey on the new button?

Reply:
No thanks -- it's rarely going to be used so don't want to use up yet another Alt+key combo on it.